ui/shared/components/form/Inputs.jsx

```diff
@@ -439,6 +439,10 @@ export const RadioButtonGroup = styled(({ radioLabelStyle, ...props }) => <Butto
       border-radius: 0;
     }
   }
+
+  &.buttons + .buttons .label {
```

Comment:
fixes a display bug when multiple RadioButtonGroup are rendered next to each other